crates/storage-query-datafusion/src/status/row.rs

```diff
@@ -86,19 +86,19 @@ fn fill_invocation_metadata(
     if let Some(deployment_id) = meta.deployment_id {
         row.pinned_deployment_id(deployment_id);
     }
-    match meta.response_sink {
-        Some(ServiceInvocationResponseSink::PartitionProcessor { caller, .. }) => {
+    match meta.caller {
+        Caller::Service(caller) => {
             row.invoked_by("service");
             row.invoked_by_service(&caller.service_id.service_name);
             if row.is_invoked_by_id_defined() {
                 row.invoked_by_id(format_using(output, &caller));
             }
         }
-        Some(ServiceInvocationResponseSink::Ingress(..)) => {
+        Caller::Ingress => {
             row.invoked_by("ingress");
         }
-        _ => {
-            row.invoked_by("unknown");
+        Caller::Internal => {
+            row.invoked_by("Restate");
```

Comment:
To match the case of ingress, maybe?
```suggestion
            row.invoked_by("restate");
```

Reply:
Will change it.